projects/batfish/src/org/batfish/grammar/cisco/Cisco_snmp.g4

```diff
@@ -67,6 +67,7 @@ ss_host_null
       INFORMS
       | TRAPS
       | VERSION
+      | variable
```

Comment:
This is a breaking change, as shown in your changes to the ref file. It seems like you are trying to parse:
`snmp-server host 10.1.2.3 xyx`
But since `variable` matches all non-`NEWLINE` tokens, and `ss_host_null` includes `~NEWLINE*`, and `ss_host_null` appears above `ss_host_use_vrf`, this will make `ss_host_use_vrf` unmatchable. Furthermore, this will cause anything after `snmp-server host` to parse, and we may fail to catch unhandled variants of currently implemented matches - even if we put `ss_host_use_vrf` above `ss_host_null`. Generally speaking, you should never put `variable` in a `xx_null` rule.